cli/js/ops/io.ts

```diff
@@ -8,12 +8,10 @@ export function readSync(rid: number, buffer: Uint8Array): number | null {
   }
   const nread = sendSyncMinimal("op_read", rid, buffer);
   if (nread < 0) {
-    throw new Error("read error");
-  } else if (nread == 0) {
-    return null;
-  } else {
-    return nread;
+    throw new Error("io: read error");
```

Comment:
Why prefix with io scope? read implies io and no other ops prefix their errors with their domain.